### DISCUSSION TITLE
suggestions for the roadmap for development

### DISCUSSION THREAD

**Discussion by Madara9744** [Madara9744/PasswordManagement#6] [Q&A] [answered]
suggestions for the roadmap for development

**Comment by rohankumardubey** [comment, marked as answer]
we can even look out for another storage system where we get added-on features like a Rest application to retrieve the passwords with an auth token.

**Reply by rohankumardubey** [reply]
found a few persistent storages XODUS, Chronicle, MapDB, and Microstream are a few of them. let's get to the use cases and choose the better choice. few are open source and few provide paid solutions. But we are anyhow inclined towards the open source solution and the latency aspect is major we can use MapDB.
